cypress/support/step-definitions/multi-action-button-steps.js

```diff
@@ -28,14 +28,14 @@ Then('Multi Action Button as on preview is {string}', (asProperty) => {
     .should('have.class', `${MULTI_ACTION_BUTTON_AS_PROPERTY}${asProperty}`);
 });
 
-Then('Multi Action Button align on preview is left', () => {
-  multiActionButtonPreview()
-    .should('not.have.class', `${MULTI_ACTION_BUTTON_ALIGN_RIGHT}`);
-});
-
-Then('Multi Action Button align on preview is right', () => {
-  multiActionButtonPreview()
-    .should('have.class', `${MULTI_ACTION_BUTTON_ALIGN_RIGHT}`);
+Then('Multi Action Button align on preview is {string}', (align) => {
+  if (align === 'left') {
```

Comment:
please add comment `// left is default align that's why it's absent inside class`

Reply:
fixed